scripts/enter-env.sh

```diff
@@ -45,10 +45,10 @@ if echo "${SETUP_DIR}" | grep -q ':'; then
 	exit 1
 fi
 
-# Check ixo-usb-jtag *isn't* install
-if [ -d /lib/firmware/ixo-usb-jtag/ ]; then
-	echo "Please uninstall ixo-usb-jtag package, the required firmware is"
-	echo "included in the HDMI2USB modeswitch tool."
+# Check ixo-usb-jtag *isn't* installed
+if [ -e /lib/udev/rules.d/85-ixo-usb-jtag.rules ]; then
+	echo "Please uninstall ixo-usb-jtag package from the timvideos PPA, the"
```

Comment:
something not right.  you cant uninstall from a repo/ppa.   I am guessing "reinstall ixo-usb-jtag  from the ppa" ?

Reply:
uninstall the thing you got from the PPA.